src/templates/student/assignment.html

```diff
@@ -103,7 +103,12 @@ <h5 class="card-title">+ {{ num_extension_runs }} Extension Runs</h5>
                           <span title="{{ commit['message'] }}">{{ commit["message"] | truncate(50, killwords=True) }}</span>
                         {% endif %}
                         <div id="grade-now-error" class="text-danger"></div>
-
+			{% if runs and feedback_url %}
+			<hr/>
```

Comment:
From [here](https://stackoverflow.com/a/3366605) it looks like the slash in the `<hr/>` isn't needed. Can we remove that to be consistent with the rest of the file?

Reply:
Fixed